controllers/wizard/fileUpload/PKPFileUploadWizardHandler.inc.php

```diff
@@ -384,7 +385,7 @@ function confirmRevision($args, $request) {
 
 		// Validate the form and revise the file.
 		if ($confirmationForm->validate($request)) {
-			if (is_a($uploadedFile = $confirmationForm->execute($request), 'SubmissionFile')) {
+			if (is_a($uploadedFile = $confirmationForm->execute(), 'SubmissionFile')) {
```

Comment:
This is for strict mode PHP warnings, right? We're probably going to have to standardize the parent `Form` class's implementation of various functions to have it require a `$request` instance, but this will take a lot of work and probably should be left out of this PR.

Reply:
Yes always for the annoying freezing state. I had the possibility to remove it because the request was not used in the execute method.